api/src/cli/load-extensions.ts

```diff
@@ -12,10 +12,10 @@ export const loadExtensions = async () => {
 
 	if (!installed) return;
 
-	try {
-		await validateMigrations();
-	} catch {
-		logger.info('Database is out of date. Skipping CLI extensions initialization.');
+	const migrationsValid = await validateMigrations();
```

Comment:
```suggestion
	const migrationsComplete = await validateMigrations();
```
I'd probably call this `migrationsComplete` instead as we're not actually checking the migrations for validity just if have have all been run or not.